examples/images_contours_and_fields/watermark_image.py

```diff
@@ -5,25 +5,21 @@
 
 Using a PNG file as a watermark.
 """
+
 import numpy as np
 import matplotlib.cbook as cbook
 import matplotlib.image as image
 import matplotlib.pyplot as plt
 
-# Fixing random state for reproducibility
-np.random.seed(19680801)
-
 
-datafile = cbook.get_sample_data('logo2.png', asfileobj=False)
-print('loading %s' % datafile)
-im = image.imread(datafile)
-im[:, :, -1] = 0.5  # set the alpha channel
+with cbook.get_sample_data('logo2.png') as file:
+    im = image.imread(file)
 
 fig, ax = plt.subplots()
 
-ax.plot(np.random.rand(20), '-o', ms=20, lw=2, alpha=0.7, mfc='orange')
+ax.plot(np.sin(10 * np.linspace(0, 1)), '-o', ms=20, alpha=0.7, mfc='orange')
```

Comment:
the actual data here doesn't really matter, so let's not bother with seeding an rng.